python/python/tests/test_async_client.py

```diff
@@ -2522,6 +2523,65 @@ async def test_zdiffstore(self, redis_client: TRedisClient):
                 await redis_client.zdiffstore("abc", ["zxy", "lkn"])
             assert "CrossSlot" in str(e)
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_zmpop(self, redis_client: TRedisClient):
+        min_version = "7.0.0"
+        if await check_if_server_version_lt(redis_client, min_version):
+            return pytest.mark.skip(reason=f"Redis version required >= {min_version}")
```

Comment:
should we do that in a function? could be very useful

Reply:
Same question as here: https://github.com/aws/glide-for-redis/pull/1418#discussion_r1604172926